website/docs/reference/node-selection/test-selection-examples.md

```diff
@@ -72,6 +72,21 @@ The "buildable" and "cautious" modes can be useful in environments when you're o
 
 </VersionBlock>
 
+<VersionBlock firstVersion="1.5" >
+
+There are three modes to configure the behavior when performing indirect selection (with `eager` as the default):
+
+1. `eager` (default) - include ANY test that references the selected nodes
+1. `cautious` - restrict to tests that ONLY refer to selected nodes
+1. `buildable` -  restrict to tests that ONLY refer to selected nodes (or their ancestors)
+1. `empty` - restrict to tests that are only for the selected node and ignore all tests from the attached nodes 
```

Comment:
as a resource, i used this https://github.com/dbt-labs/dbt-core/blob/87ea28fe84edd65d589fd3cf658d8dd1f65c000a/core/dbt/graph/selector.py#L219 .

not sure if i interpreted it correctly tho.